doc/sailfish-webview-webview.qdoc

```diff
@@ -66,6 +68,505 @@
+
+  Clients can set this property to navigate to a new location.
+  If the webview is redirected, the value of this property will automatically
+  be updated.
+*/
+
+/*!
+  \qmlproperty QString WebView::title
+  \readonly
+  \brief The page title of the currently loaded location
+*/
+
+/*!
+  \qmlproperty bool WebView::canGoBack
+  \readonly
+  \brief Whether the "back" navigation action is available
+
+  If the user has previously navigated to a different page via setting the
+  \l{url} property, then back navigation will be available.
+
+  \sa goBack()
+*/
+
+/*!
+  \qmlproperty bool WebView::canGoForward
+  \readonly
+  \brief Whether the "forward" navigation action is available
+
+  If the user has previously navigated back (via \l{goBack()}) without later
+  loading a new page via setting the \l{url} property, then forward navigation
+  will be available.
+
+  \sa goForward()
+*/
+
+/*!
+  \qmlproperty int WebView::loadProgress
+  \readonly
+  \brief The load progress (percentage) of the current page
+
+  When the webview begins loading a new page, this value will be set
+  to 1 (percent) and then updated as more content is loaded.  When loading is
+  complete, this value will be set to 100 (percent).
+*/
+
+/*!
+  \qmlproperty bool WebView::loading
+  \readonly
+  \brief Whether the webview is currently loading page content
+
+  \sa loadProgress, loaded
+*/
+
+/*!
+  \qmlproperty QRectF WebView::contentRect
+  \readonly
+  \brief The bounding box rectangle defined by the loaded content
+         being displayed in the viewport of the webview.
+
+  The loaded page content is likely to fill an area much larger than the
+  viewport of the webview (thus requiring the user to scroll the viewport in
+  order to see content further down the page, etc).
+
+  This property provides the bounding box rectangle of the current viewport
+  onto the content, in CSS pixels (that is, not yet scaled by the device
+  resolution factor).  The x and y position of the content rectangle describes
+  the top-left pixel of the content being displayed in the viewport.  The
+  width and height of the content rectangle can be combined with the x and y
+  positions to determine the top-right, bottom-left, and bottom-right pixels
+  of the content being displayed in the viewport.
+
+  Note that the content width and content height will differ from the
+  viewport width and viewport height only if some zoom-factor is being applied
+  to the viewport, or if the content is more narrow than the viewport.
+
+  \sa contentWidth, contentHeight, viewportWidth, viewportHeight, scrollableSize
+
+  // TODO: is this correct?  Have I understood this property correctly?
+  //       is this in device pixels or in CSS pixels?
+  //       it appears to be CSS pixels, since these values are scaled by the
+  //       contentResolution before updating the scrollableSize etc?
+  //       if so, how should the documentation here be updated/changed?
+  //       how is the zoomFactor defined?
```

Comment:
I think you covered nicely already the contentRect

Reply:
thanks, removed the comments now, and added some context links to Mozilla docs as suggested by David.